src/components/Favorites/Favorite.component.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { List } from 'immutable';
+import propTypes from 'prop-types';
+
+import LinkToSongContainer from '../Files/LinkToSong';
+
+const FavoriteComponent = ({ favorites }) => (
+  <div className="content-container">
+    <h1 id="your-personal-library" className="title">
+      Your personal library
+      <span role="img" aria-label="heart">❤️</span>
```

Comment:
aria-label value does not make sense

Reply:
Based on this documentation (https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/accessible-emoji.md) it seems this is the correct way of naming it. If not, what do you suggest naming it? 🤔

Reply:
Nvm, for some reason I misread this. I thought this was a button or something alike (?)